features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x/NRFCordioHCIDriver.cpp

```diff
@@ -298,6 +300,7 @@ void NRFCordioHCIDriver::do_initialize()
 
     // For some reason, the mbed target uses this (TIMER0_IRQHandler_v) vector name instead of the "standard" TIMER0_IRQHandler one
     NVIC_SetVector(TIMER0_IRQn, (uint32_t)TIMER0_IRQHandler);
+    NVIC_SetVector(TIMER2_IRQn, (uint32_t)TIMER2_IRQHandler);
```

Comment:
Please update the comment. This is the case for 2 IRQs and a note for future porters might be useful.